scripts/globals/mobskills/murk.lua

```diff
@@ -19,6 +18,7 @@ end
 mobskill_object.onMobWeaponSkill = function(target, mob, skill)
     local slowed = false
     local weight = false
+    local typeEffect
```

Comment:
Quit coding tired Teo, this var isn't getting used!

Reply:
yes it is!

its initialized nil, then changing in two different conditions below.